scRNA-seq/02-tag-based_pre-processing_scRNA-seq.md

```diff
@@ -19,12 +21,10 @@ Salmon, just like was done in the bulk RNA-seq module.
 
 ### Step 0: Obtaining the data
```

Comment:
Can we make this `Data` or something similar? There is no obtaining (e.g., use of `wget`) that occurs here.

Reply:
Right.

Reply:
Changed to "About the data"